src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -31,8 +31,22 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// <see cref="ColumnCopyingEstimator"/> copies the input column to another column named as specified in the parameters of the transformation.
+    /// Estimator creating the <see cref="ColumnCopyingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ### Estimator FAQ
+    /// | Question | Answer |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type. | Any |
+    /// | Output column data type. | The same as the data type in the input column. |
```

Comment:
maybe we can add a new row for "Associated transformer"?

Reply:
? you mean the transformer that gets generated from Fit?

it is right below the table. 

---
In reply to: [275070613](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275070613) [](ancestors = 275070613)